package.json

```diff
@@ -3,9 +3,9 @@
   "publishConfig": {
     "access": "public"
   },
-  "version": "4.1.2",
+  "version": "4.5.0",
```

Comment:
IMO, The problem with this approach is that the users who want to install a specific version, say, `4.2.0`, they can't because no release exists. So the better way is always to fetch the required version, or setup the CI, which will bump up the version when it's available, and do an automatic release

Reply:
We can do that in a future bit of work. The effort for setting up an automated task is going to be way higher than manually bumping this once a month, which is all we need to do. 

I also don't think people want a specific version, we have had this for a while and no one has asked "How do I install this specific version?".